microservices/clients/contestproposal-client/src/main/java/org/xcolab/client/proposals/ProposalAttributeClientUtil.java

```diff
@@ -23,11 +23,6 @@ public static ProposalAttribute createProposalAttribute(
         return client.createProposalAttribute(proposalAttribute);
     }
 
-    public static ProposalAttribute getImpactProposalAttributes(
-            Long proposalId) {
-        return client.getImpactProposalAttributes(proposalId);
-    }
-
```

Comment:
This change also seems unrelated.

Reply:
You can ignore this, the change has no functional impact as the method was unused.